__tests__/app.test.js

```diff
@@ -103,3 +103,37 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: should return an array or article objects (with body property) in descending order by created_at", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(13);
+        expect(typeof body).toBe("object");
+        expect(Array.isArray(body.articles)).toBe(true);
+        body.articles.forEach((article) =>
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+          })
+        );
+        expect(body.articles[12]).toEqual({
```

Comment:
This extra assertion does work but I'd say isn't required since you've covered the object structure and data types in the test above. In this case it does work as the articles are always inserted in the same order but just something to be aware of as changing a sortby in your DB query could cause this test to break.